TemperatureControl/TemperatureControl.js

```diff
@@ -0,0 +1,169 @@
+/**
+ * Provides Agate-themed temperature control components and behaviors.
+ *
+ * @example
+ * <TemperatureControl
+ *   max={30}
+ *   min={10}
+ *   unit="Celsius"
+ *   value={10}
+ * />
+ *
+ * @module agate/TemperatureControl
+ * @exports TemperatureControl
+ * @exports TemperatureControlBase
+ * @exports TemperatureControlDecorator
+ */
+
+import kind from '@enact/core/kind';
+import Changeable from '@enact/ui/Changeable';
+import Pure from '@enact/ui/internal/Pure';
+import MeasurementFactory from 'ilib/lib/MeasurementFactory';
+import UnitFmt from 'ilib/lib/UnitFmt';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+import React from 'react';
+
+import ArcSlider from '../ArcSlider';
+import Skinnable from '../Skinnable';
+
+import css from './TemperatureControl.module.less';
+
+/**
+ * Temperature control base component.
+ *
+ * @class TemperatureControlBase
+ * @memberof agate/TemperatureControl
+ * @ui
+ * @public
+ */
+const TemperatureControlBase =  kind({
+	name: 'TemperatureControl',
+
+	propTypes: /** @lends agate/TemperatureControl.TemperatureControlBase.prototype */ {
+		/**
+		 * The minimum value of the slider.
+		 *
+		 * @type {Number}
+		 * @default 30
+		 * @public
+		 */
+		max: PropTypes.number,
+
+		/**
+		 * The minimum value of the slider.
+		 *
+		 * @type {Number}
+		 * @default 10
+		 * @public
+		 */
+		min: PropTypes.number,
+
+		/**
+		 * Called when value is changed.
+		 *
+		 * @type {Function}
+		 * @public
+		 */
+		onChange: PropTypes.func,
+
+		/**
+		 * The unit of the temperature(Celsius or Fahrenheit).
+		 *
+		 * @type {('Celsius'|'Fahrenheit')}
+		 * @default 'Celsius'
+		 * @public
+		 */
+		unit: PropTypes.oneOf(['Celsius', 'Fahrenheit']),
```

Comment:
I think the default value is `Celsius` but I'm seeing Fahrenheit in sampler when it loads.
![image](https://user-images.githubusercontent.com/5037429/95183063-686e1880-0800-11eb-86cd-2d8df4d8dba9.png)

If I change the value as `Fahrenheit`, the unit is not changing and the value seems `Celsius`.
![image](https://user-images.githubusercontent.com/5037429/95183151-88054100-0800-11eb-8315-f241d75fb679.png)

Reply:
Yes, this is because of the ilib library which converts the value based on locale. The unit is provided and the value. If the provided is 15Celsius and the current locale us en-us, it will convert 15 Celsius to the Fahrenheit Value.
If the provided value is 15F and locale is en-us, no covnersion will be made.

`const currentTemperature = MeasurementFactory({unit, amount: value});`

Reply:
I see. I saw @ybsung suggested this way to you. I'm sorry but I think it's a bit confusing to users.
We had a discussion with members and we think it'd be better not to have this conversion based on the locale.
So that we could avoid this kind of confusing and the component will just "display" the input.
Since we don't have any specific UX requirements, we could delegate this locale-based conversion to apps.
I'm sorry but could you remove this conversion from your PR? Thank you.

Reply:
cool :) I'll solve the conflicts right away